### DISCUSSION TITLE
Release 2.1.5

### DISCUSSION THREAD

**Discussion by Altonss** [grote/Transportr#854] [General]
In this thread we can gather the discussion for the next small 2.1.5 release.

For this release, these steps need to be done:

- [x] Merge https://github.com/grote/Transportr/pull/853
- [x] Merge https://github.com/grote/Transportr/pull/850 ? (seems very easy)
- [x] Maybe also merge #856?
- [ ] Write release notes for the release (check which bugs and networks are fixed)
- [ ] Update PTE and follow all the other steps of this template for the final PR https://github.com/grote/Transportr/pull/736
- [ ] Publish new version (github + Playstore)

**Comment by Altonss** [comment]
One additionnal thing, would maybe to update the app website (and maybe also the banner?) on the PlayStore: https://play.google.com/store/apps/details?id=de.grobox.liberario
Also at least in french, the title of the app seems to be cut off, so maybe it should be made shorter?

**Reply by Echolon** [reply]
https://github.com/grote/Transportr/issues/857

**Comment by genofire** [comment]
Please merge the PR to bring the Transportr back to F-Droid:
see #826 and #835

**Reply by Altonss** [reply]
There isn't a ready PR that would bring back Transportr back to F-Droid. You linked to one issue and a small PR that doesn't fix the issue with the maplibre branch. This still needs some work before getting back to F-Droid :)
And this work would probably only be done after the small 2.1.5 maintenance release.

**Comment by grote** [comment]
PTE update is missing from the list above it seems.

**Reply by Altonss** [reply]
I considered included in :
>    -  [ ]  Follow this template for the final PR [Release 2.1.1 #736](https://github.com/grote/Transportr/pull/736), so 2.1.5 can be release

but will rephrase it :)

**Comment by Echolon** [comment]
# FINAL DRAFT
## Release Notes Transportr 2.1.5

Dear Transportr user,

Its been more than two long year since the last update. We still hope this app have you been serving well so far when you read this. But we also understand if you were wondering what is going on. If you are new to the app - a warm welcome and great to have you here! We have provided more information about the future development of Transportr: https://github.com/grote/Transportr/issues/852

Thanks to the contributors of this update!
@grote
@ialokim
@Altonss 
@pt2121
@thgoebel
@mimi89999 

## Changes in Transportr 2.1.5

We have mostly maintenance changes in this update but it should solve many longstanding issues!

- Add themed launcher Icons.
- Fix a bug where calendar and map apps are not found.
- Update the Public Transport Enabler (PTE) data library - remove transport networks VMS, VRS, and PL.
- Update the list of networks in Poland.

**Reply by Altonss** [reply]
Thanks a lot for the work!
There a 2 questions coming to my mind:
1) Do we want to explain all this to the user (everything else then changes)?
2) Do we have enough room to do it in the app/appstores?

These are open questions, so feel free to comment :)

------
>     * Ensure Transportr is availabe in F-Droid Store again
This will probably not be part of 2.1.5 as it requires a lot of work...

**Reply by Echolon** [reply]
Yes, I understand. I think we should send some message. Or ask @grote if we can open a blog section on the webpage and link to it.

But I think its a way to reach many interested users. Also to spread the word and give some background. People should understand at least a bit.

**Reply by Echolon** [reply]
Btw this is what I want to show at least in the in-app overlay

**Reply by pt2121** [reply]
I'm preparing for the release #861 but haven't used the changelog from here - I'm waiting for a final version. In my opinion, the end users don't need to know about SDK changes. What do you think?

I am thinking:

- Add themed launcher Icons.
- Fix a bug where calendar and map apps are not found.
- Update the PTE data library - remove VMS, VRS, and PL.
- Update the list of networks in Poland.

**Reply by Altonss** [reply]
The PL network was poland? It yes, it's strange to say in the changelog that it got removed, and the list in poland got updated :thinking: 
Also, for the changelog is presented to users, I would add a small detail that VMS, VRS and PL are networks (or transport networks). Idk if we should include the PTE acronym?
Otherwise this LGTM :)

**Reply by pt2121** [reply]
I  don't have a strong opinion on this so I'll let y'all decide, cc @Echolon. I'll copy and paste the final one to the release PR. (Hope we'll find a way/process to do it without Copy & Paste in the next release)
ty!

**Reply by Echolon** [reply]
Ok, what does PTE stand for?

I updated the other stuff according to suggestions and shrinked the text again.

@grote @ialokim any concerns?

**Reply by Altonss** [reply]
PTE = Public Transport Enabler

**Reply by ialokim** [reply]
I think I agree with @Altonss in that we should not overload the changelog with information. The four points regarding maintenance changes in this version should be included, and maybe a first one that points to some information we could add to the website? Feel free to look open a respective PR on the website repo: https://github.com/grote/Transportr-website

This text could definitely include more information.

**Reply by Echolon** [reply]
Final draft is up. Can we quickly create a blogpost on the website, see https://github.com/grote/Transportr-website/issues/14 ? Any more content to add?

**Reply by Echolon** [reply]
Task 4 is done.

**Reply by pt2121** [reply]
I can see one confusion now. PL is a network id of **Poland, railways in Poland**.
We still have another network in Poland called **Municipal Transports of Poland**. So, we haven't completely removed Poland.

**Comment by mgw2013** [comment]
Changed to github app now and things are working. Question: how to report issues with maps/bus stops missing?

**Reply by Altonss** [reply]
I'm happy to hear it now works for you :) (We still plan to bring Transportr back to F-Droid)
What do you mean with maps/bus stops missing? You mean on the map displaying or on the route displayed?

**Comment by mgw2013** [comment]
The inital stop is shown somewhere else (bus storage area, I presume). First station (start position of route) not shown. Might be a data source issue...?

**Reply by Altonss** [reply]
Please open an issue at https://github.com/grote/Transportr/issues.
Aditionnaly we would need:
- clear explanation (with start and stop of route if possible)
- Transportr version
- network selected
- Android version if possible
- A screenshot would be very useful!

**Comment by grote** [comment]
Released to Google Play Beta track now (under review still). Please let me know when ready for production.

It also got release to my F-Droid repo. Official fdroid still pending map library change.

**Reply by Echolon** [reply]
Ref: https://github.com/grote/Transportr/pull/880
